obspy/io/seiscomp/tests/test_event.py

```diff
@@ -59,7 +58,11 @@ def cmp_read_xslt_file(self, sc3ml_file, quakeml_file, validate=True):
             if validate:
                 self.assertTrue(_validate_quakeml(tf.name))
             filepath_cmp = os.path.join(self.path, quakeml_file)
-            self.assertTrue(filecmp.cmp(filepath_cmp, tf.name))
+            tf.seek(0)
+            dat1 = tf.read().decode("utf-8").splitlines()
+            with open(filepath_cmp, 'rb') as f:
+                dat2 = f.read().decode("utf-8").splitlines()
+            self.compare_list_of_lines(dat1, dat2)
```

Comment:
Sorry to nitpick here - how about a _compare_files(filename, tf) function? That would even further deduplicate the code.

Reply:
indeed :) I think you have access to the repo, don't you ? :)

Reply:
fair enough ;)